borrowings/serializers.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+
+from books_service.models import Book
+from books_service.serializers import BookSerializer
+from borrowings.models import Borrowing
+
+
+class BorrowingListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Borrowing
+        fields = "__all__"
```

Comment:
you should show book info here as well using the book serializer, otherwise, we'll see only the book's id, which is not pretty